README.md

```diff
@@ -144,6 +144,10 @@ configuration passed to the route.
 
 Object with [reply options](https://github.com/fastify/fastify-reply-from#replyfromsource-opts) for `fastify-reply-from`.
 
+### schema
+
+This option is the same as [specified here](https://www.fastify.io/docs/latest/Routes/#routes-options). Note that certain validations may not work properly depending on the proxy settings used - e.g. `body` - since payloads may be directly streamed.
```

Comment:
I'm not sure about this feature because it would mean that, every proxied route would share the same schemas, so `querystring` checks will run even per `POST` route that, usually is not wanted.

Could we think of another approach for this feature?

Reply:
IMO that's a userland problem - someone who wants to avoid that needs to scope their different routes/schema under different `prefix` - or they can manually set this up using `fastify-reply-from`

I'm ofc happy to disclaim that problem in the README.

Otherwise, just the user-options required to support what you're describing seems a bit complex to me - it'd be something like `opts.post.schema`? And then each METHOD would have to have its own declaration?

Reply:
My suggestion of declaring each route individually seems like it'd be a breaking change to `reply.context.config`?

This assertion fails if we change from declaring `method` in bulk to individually:

```
      async preHandler (request, reply) {
        t.deepEqual(reply.context.config, {
          foo: 'bar',
          url: '/*',
          method: [
            'DELETE',
            'GET',
            'HEAD',
            'PATCH',
            'POST',
            'PUT',
            'OPTIONS'
          ]
        })
```